Tests fail due to missing `django.utils.six`

```
______________ ERROR collecting tests/test_management_commands.py ______________
ImportError while importing test module '/home/travis/build/Styria-Digital/django-rest-framework-jwt/tests/test_management_commands.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
.tox/py36-dj30-drf311-codecov/lib/python3.6/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_management_commands.py:7: in <module>
    from django.utils.six import StringIO
E   ModuleNotFoundError: No module named 'django.utils.six'
```

This got removed in Django 3.0 and should be replaced with `six`. See also https://stackoverflow.com/a/59252857/320546

--- a/setup.cfg
+++ b/setup.cfg
@@ -54,6 +54,7 @@ test =
     pytest-django
     pytest-runner
     cryptography==2.0.3
+    six
 docs =
     mkdocs==0.13.2
 

--- a/tests/test_management_commands.py
+++ b/tests/test_management_commands.py
@@ -4,7 +4,7 @@ from __future__ import unicode_literals
 
 from django.conf import settings
 from django.core.management import call_command
-from django.utils.six import StringIO
+from six import StringIO
 from pytest import raises
 from rest_framework_jwt.authentication import JSONWebTokenAuthentication
 

